app/assets/stylesheets/colors.scss

```diff
@@ -18,6 +18,8 @@ $footer-gray: #333;
 
 
 $highlight: 0 1px 0 $white;
+$orange_highlight: #fc9800;
```

Comment:
Name of variable `orange_highlight` should be written in all lowercase letters with hyphens instead of underscores